src/skvhttpclient/test/ClientIntegrationTest.cpp

```diff
@@ -526,6 +526,21 @@ void testReadWriteConflict() {
   }
 }
 
+void testDropCollection() {
```

Comment:
Please also add a test to validate that we can create a new collection with the same name after the old one is dropped

Reply:
Done